### PR TITLE
Rework some things in the Wiki related to Optifine and alternatives

### PR DESCRIPTION
**Pull request by flowln**
This makes some formatting more consistent, fix some types and add some explanations as to why you should use Optifine alternatives.

### CODE REVIEW

File: src/wiki/getting-started/install-of-alternatives.md [hunk 1]
```diff
@@ -5,31 +5,31 @@ eleventyNavigation:
   order: 7
 ---
 
-[Make sure to have Fabric API](../download-mods/#fabric)
+Make sure to have [Fabric API](../download-mods/#fabric), as many Fabric mods depend on it to work.
```
Comment (DioEgizio) [inline src/wiki/getting-started/install-of-alternatives.md:8]:
```suggestion
NOTE: some mods require [Fabric API](../download-mods/#fabric), to work.
```
Wouldn't this be better?

File: src/wiki/getting-started/installing-optifine.md [hunk 2]
```diff
@@ -6,30 +6,34 @@ eleventyNavigation:
...
+Remember where you have kept your .jar and continue to the PolyMC launcher. Follow the steps from [download-mods](../download-mods) and choose either forge or fabric([!warning](#fabric)).
 
 ## Forge
-Forge does not require exta steps besides adding the .jar for optifine. 
+Forge does not require exta steps besides [adding the .jar for optifine](#adding-.jar-file). 
 
 NOTE: some OF versions **don't** work on Forge!
 
 ## Fabric 
 [Requires Fabric API](../download-mods/#fabric)
```
Comment (DioEgizio) [inline src/wiki/getting-started/installing-optifine.md:30]: Optifabric doesn't require FAPI but whatever

**Review by DioEgizio — CHANGES_REQUESTED** [review]
Other than this, LGTM

File: src/wiki/getting-started/installing-optifine.md (hunk 2 above)
Comment (flowln) [reply src/wiki/getting-started/installing-optifine.md:30]: Well they do say this in their wiki:
image
Though it is from 2019 so idk

Comment (DioEgizio) [reply src/wiki/getting-started/installing-optifine.md:30]: 2019 Optifabric was pretty different
Not listed as a dependency on cf nor on https://github.com/Chocohead/OptiFabric/blob/llama/src/main/resources/fabric.mod.json

Comment (flowln) [reply src/wiki/getting-started/installing-optifine.md:30]: I'll just add a warning for versions < 1.16 then, as some people might run into problems without it

Comment (DioEgizio) [reply src/wiki/getting-started/installing-optifine.md:30]: ok then

**Review by DioEgizio — APPROVED** [review]

**Pull request merged**